data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileCracCreatorTest.java

```diff
@@ -72,6 +106,7 @@ public void testCreateCracCSATestConfigurationTC2Draftv14Apr2023() {
 
         assertNotNull(cracCreationContext);
         assertTrue(cracCreationContext.isCreationSuccessful());
+        assertEquals(10, cracCreationContext.getCreationReport().getReport().size());
         assertEquals(15, cracCreationContext.getCrac().getContingencies().size());
```

Comment:
could you please add here assertEquals crac.getFlowCnecs.getSize to directly understand the number of entries in the report ?